### ISSUE TITLE
Fix JWT formation with ACL parameter

### ISSUE THREAD

**Issue by hummusonrails** [Nexmo/nexmo-jwt-ruby#9]
Currently, the JWT gem is adding an extra top-level parameter to the token when a user adds ACL information. 

The relevant part of the token's structure looks like this right now:

```ruby
"application_id": "xxxxxxxxxxx",
  "paths": {
    "acl": {
      "paths": {
        "/*/users/**": {},
        "/*/conversations/**": {},
      }
    }
  }
```

This is incorrect.

It should look like:

```ruby
"application_id": "xxxxxxxxx",
  "acl": {
    "paths": {
      "/*/users/**": {},
      "/*/conversations/**": {},
    }
  }
```

The code that is impacting this can be found at [https://github.com/Nexmo/nexmo-jwt-ruby/blob/master/lib/nexmo-jwt/jwt.rb#L26](https://github.com/Nexmo/nexmo-jwt-ruby/blob/master/lib/nexmo-jwt/jwt.rb#L26):

```ruby
hash.merge!(paths: generator.paths) if generator.paths
```

A full sample JWT payload for Vonage can be found on the [Vonage API Developer Portal](https://developer.nexmo.com/conversation/guides/jwt-acl#sample-jwt-payload) along with more information on the structure.

The solution to this issue needs to also have additional tests added to validate both the success of the change and to confirm the fail route as well (i.e. Does it work properly now and does it explicitly _not_ create the structure incorrectly anymore?).

**Comment by ananichev** [comment]
Hey @bencgreenberg, could you please review my PR https://github.com/Nexmo/nexmo-jwt-ruby/pull/10

**Comment by hummusonrails** [comment]
Hi @ananichev. I'll be taking a look today. Thanks for the PR!

**Comment by hummusonrails** [comment]
Addressed with #10
